packages/composites/proof-of-reserves/src/endpoint/reserves.ts

```diff
@@ -38,13 +39,14 @@ const inputParameters: InputParameters = {
 
 export const execute: ExecuteWithConfig<Config> = async (input, context, config) => {
   const validator = new Validator(input, inputParameters, paramOptions)
-
   const jobRunID = validator.validated.jobRunID
   const protocol = validator.validated.data.protocol.toUpperCase()
   const indexer: Indexer = validator.validated.data.indexer.toUpperCase()
   const confirmations = validator.validated.data.confirmations
 
   const protocolOutput = await runProtocolAdapter(jobRunID, context, protocol, input.data, config)
+  protocolOutput.result = validateAddresses(indexer, protocolOutput.result)
```

Comment:
We should have a way to disable address validation, probably with an input param.

Reply:
Or... have a way to do address validation, but still allow duplicate addresses. I just want us to think ahead and not have to do code changes in the future if we suddenly want duplicate addresses for some use case.

Reply:
Added option to disable validation altogether & added param info to readme